src/test/java/app/services/impl/GameServiceImplTest.java

```diff
@@ -0,0 +1,135 @@
+package app.services.impl;
+
+import app.dtos.GameDTO;
+import app.dtos.GenreDTO;
+import app.dtos.ParentPlatFormDTO;
+import app.dtos.PlatformDTO;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameServiceImplTest {
+
+    private static GameServiceImpl gameService;
```

Comment:
Should this be `GameService gameService` instead of `GameServiceImpl`? Even though we're testing the implementation it's still primarily used with interface and then whatever implementation can be stored in it. but an argument could also be made to use the implementation class. Food for thought

Reply:
Agree that the object should be of type GameService since we do inherit from that interface